spaceapi_server/src/lib.rs

```diff
@@ -98,11 +115,17 @@ impl SpaceapiServer {
 
 impl middleware::Handler for SpaceapiServer {
 
-    fn handle(&self, _: &mut Request) -> IronResult<Response> {
+    fn handle(&self, req: &mut Request) -> IronResult<Response> {
+
+        match req.get_ref::<UrlEncodedQuery>() {
+            Ok(ref hashmap) => return self.update_values(hashmap),
+            Err(ref e) => println!("{:?}", e)
+        };
```

Comment:
So right now the same endpoint is used for getting and updating values? Maybe we can refactor that into separate endpoints later on :)